src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(a, b) {
+      return a + b;
+    },
+
+    subtract(a, b) {
+      return a - b;
+    },
```

Comment:
```suggestion
    add(a, b) {
      return a + b;
    },

    subtract(number) {
      return this.result - number;
    },
```
Try to make it work like that